src/data-dictionary/events/NrAiIncident/closeCause.md

```diff
@@ -5,21 +5,28 @@ events:
   - NrAiIncident
 ---
 
-If applicable, what caused the incident to close.
-
 Available values: 
+
```

Comment:
left similar note on an event: we want to avoid these long attribute definitions: this seems like something we should have in the docs and not here.